.github/workflows/setup_venvs.yaml

```diff
@@ -0,0 +1,75 @@
+name: Setup virtual environments
+on: 
+  workflow_call:
+    inputs:
+      os:
+        required: true
+        type: string
+        default: ubuntu-latest
+  # workflow_dispatch:
+  # workflow_run:
+  #   workflows:
+  #     - Setup basic software environment
+  #   types:
+  #     - completed
+  # push:
+  # pull_request:
+  #   types: [review_requested, ready_for_review]
+
+jobs:
+
+  Setup_software_basis_py310_latest:
+    uses: ./.github/workflows/setup_basic_software.yaml
+    with:
+      os: ubuntu-latest
+      
+      # with:
+      #   cache-name: "cache-software-basis"
```

Comment:
I'm guessing these lines are commented out because the cache is already specified in the `setup_basic_software` workflow? In that case these lines can probably be removed here.

```suggestion
```